### DISCUSSION TITLE
API Include ShortURL Tag When Creating ShortURL

### DISCUSSION THREAD

**Discussion by TritonB7** [shlinkio/discussions#93] [Help wanted] [answered]
I have a file and pastebin service that uses Shlink to shorten the public facing URLs. Currently when generating short links within these services, Shlink will create the short url - however the title and/or tags are of course blank on Shlink's backend. Is there anyway to populate the tags automatically or include them when generating the short URL? 

I'm currently using something like this for the services:

"https://doma.in/rest/v3/short-urls/shorten?apiKey=00000000-0000-0000-0000-000000000000&format=txt&longUrl="

**Comment by acelaya** [comment, marked as answer]
Hello @TritonB7 

The usage of that endpoint is discouraged, and it is only suitable for certain integrations for non-configurable third parties.

In order to be able to provide extra options (title, tags, metadata, etc) you should use this one https://api-spec.shlink.io/#/Short%20URLs/createShortUrl
